extensions/amp-inputmask/0.1/test/integration/test-integration-amp-inputmask-date.js

```diff
@@ -16,7 +16,7 @@
 
 import {simulateKeyboardInteraction} from './utils';
 
-const config = describe.configure().retryOnSaucelabs().ifChrome();
+const config = describe.configure().ifChrome();
```

Comment:
For my understanding, is `ifChrome` still needed if tests are all run on local Chrome?

Reply:
`ifChrome()` ensures this test runs only on Chrome, and will prevent it from running on other browsers, as originally intended.